src/223/main/kotlin/org/rust/ide/inspections/import/RsReferenceImporter.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections.import
+
+import com.intellij.codeInsight.daemon.ReferenceImporter
+import com.intellij.openapi.editor.Editor
+import com.intellij.psi.PsiFile
+import org.rust.ide.inspections.RsUnresolvedReferenceInspection
+import org.rust.ide.settings.RsCodeInsightSettings
+import org.rust.lang.core.psi.RsFile
+import org.rust.lang.core.psi.RsMethodCall
+import org.rust.lang.core.psi.RsPath
+import org.rust.lang.core.psi.ext.RsElement
+import org.rust.lang.core.psi.ext.containingCrate
+import org.rust.lang.core.psi.ext.isUnderCfgTest
+import java.util.function.BooleanSupplier
+
+class RsReferenceImporter : ReferenceImporter {
+    override fun autoImportReferenceAtCursor(editor: Editor, file: PsiFile): Boolean = false
+
+    @Suppress("UnstableApiUsage")
+    override fun computeAutoImportAtOffset(editor: Editor, file: PsiFile, offset: Int, allowCaretNearReference: Boolean): BooleanSupplier? {
```

Comment:
Could you check a platform version when this API appeared and use it as minor supported version for 223 builds?